### PR TITLE
Fix grammar in docs: communication-in-microservice-architecture.md

### PR DESCRIPTION
**Pull request by jjhampton**
# Title

Fix grammar in communication-in-microservice-architecture.md

## Summary

Rephrase 'not' => 'Neither' + 'nor'

### CODE REVIEW

File: docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ The second axis is defining if the communication has a single receiver or multip
...
 
-These axes are good to know so you have clarity on the possible communication mechanisms, but they are not the important concerns when building microservices. The asynchronous nature of client thread execution not even the asynchronous nature of the selected protocol are the important points when integrating microservices. What *is* important is being able to integrate your microservices asynchronously while maintaining the independence of microservices, as explained in the following section.
+These axes are good to know so you have clarity on the possible communication mechanisms, but they are not the important concerns when building microservices. The asynchronous nature of client thread execution nor even the asynchronous nature of the selected protocol are the important points when integrating microservices. What *is* important is being able to integrate your microservices asynchronously while maintaining the independence of microservices, as explained in the following section.
```
Comment (mairaw) [inline docs/standard/microservices-architecture/architect-microservice-container-applications/communication-in-microservice-architecture.md]: I think the sentence would read better like the following:
"Neither the asynchronous nature of client thread execution nor the asynchronous nature..." (so add neither and remove even)

**Review by mairaw — CHANGES_REQUESTED** [review]
Good catch @jjhampton! I've left some feedback to improve the sentence a bit more. 

Ping @CESARDELATORRE since this is in the e-book's content.

**Comment by jjhampton** [comment]
@mairaw Thanks for the suggestion, I agree w/ the phrasing you proposed for that sentence. I've added the requested changes and force-pushed to the remote branch.
*ping* @CESARDELATORRE

**Review by mairaw — APPROVED** [review]
Thanks @jjhampton for making the suggested changes! It looks great. Your fix should be live in the next day or so.

**Pull request merged**